estate_account/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+{
+    'name': "Link Real Estate - Accounting",
+    'depends': ['base', 'account', 'estate'],
```

Comment:
```suggestion
    'depends': ['account', 'estate'],
```
not needed because the others already depends on base